outline.md

```diff
@@ -0,0 +1,44 @@
+# Learning Outline 
+
+## Content
+
+The workshop is divided into the following three sections:
+
+### The Unix Shell
+
+The aim of this section is not to teach learners basic unix commands,
+or syntax for bash scripting,
+but rather to motivate them
+to **automate tasks and develop pipelines**.
+Topics covered will include filesystem navigation and manipulation,
+redirection and piping of standard input and output,
+file permissions, and shell scripts.
+
+### Cluster structure and scheduling
+
+This section will be very brief. It is meant to convey a simplistic mental model of the cluster,
+and how tasks get submitted, assigned and executed on the cluster. The used HPC infrastructure
+
+At the end of the section, they will
+submit a number of "Hello World" style batch jobs that are aligned to the Unix Shell section above.
+
+### Parallel workflows
+
+This section will guide learners through the process of
+performing a large HPC simulation (code available),
```

Comment:
Is having an available code enough? Most HPC sites use a module environment to deliver software, so won't we have to go into how to use the modules tool.